### PR TITLE
feat(service): Add Mircosoft Teams service

### PR DESCRIPTION
**Pull request by nikoksr**
### Info

Adding support for [Microsoft Teams](https://www.microsoft.com/en-US/microsoft-teams/group-chat-software). With this change you will be able to send messages to multiple MS-Teams channels via their webhook.

See #8 for more.

### Proposed changes

- Add **ms_teams** service
- Add notes about **ms_teams** to ReadMe

### Verification

I personally don't use Microsoft Teams so unfortunately I can't verify the functionality of this service. Maybe someone who has access to such a channel could verify if sending messages works as intended? I will not merge until we have confirmation.

### CODE REVIEW

**Comment by j-wil** [comment]
@nikoksr I use teams often I'll test this and report back

File: service/msteams/ms_teams.go [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+func New(apiToken string) (*MSTeams, error) {
+	client := goteamsnotify.NewClient()
+
+	m := &MSTeams{
+		client:   client,
+		webHooks: []string{},
+	}
+
+	return m, nil
+}
```
Comment (renanbastos93) [inline service/msteams/ms_teams.go:26]: in this method, you don't validate the error then we can change the signature method
```suggestion
func New(apiToken string) *MSTeams {
	client := goteamsnotify.NewClient()

	m := &MSTeams{
		client:   client,
		webHooks: []string{},
	}

	return m
}
```

**Review by renanbastos93 — CHANGES_REQUESTED** [review]
I believe that is necessary to improve the method new, to later thinking to merge it.

File: service/msteams/ms_teams.go [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+// MSTeams struct holds necessary data to communicate with the MSTeams API.
+type MSTeams struct {
+	client   goteamsnotify.API
+	webHooks []string
+}
+
+// New returns a new instance of a MSTeams notification service.
+// For more information about telegram api token:
+//    -> https://github.com/atc0005/go-teams-notify#example-basic
+func New(apiToken string) *MSTeams {
```
Comment (j-wil) [inline service/msteams/ms_teams.go:17]: @nikoksr `apiToken` is not required here all authentication for this service is done using the webhook urls which gets injected using the `AddReceivers` function.

**Review by j-wil — CHANGES_REQUESTED** [review]
Looks good and code works if you fix the one comment I had.

File: service/msteams/ms_teams.go (hunk 2 above)
Comment (nikoksr) [reply service/msteams/ms_teams.go]: Good catch!

**Comment by nikoksr** [comment]
Closes #8

**Pull request merged**